File: python/marks.py
maths = int(input("maths marks "))
english = int(input("engish marks "))
science = int(input("science marks "))

average = (maths+english+science)/3

if average >= 90 :
    print("O")
elif average >= 80 : 
    print("A+")
elif average >= 70 :
    print("B+")
elif average >= 60 :
    print("B")
else : 
    print("C")

File: java/winter-pep/14-01/Marks.java
public class Marks {
    public static void main(String[] args) {
        String str = "English 50 Science 76 Maths 90";
        String[] arr = str.split(" ");

        int sum = 0;

        for (int i = 1; i < arr.length; i += 2) {
            sum += Integer.parseInt(arr[i]);
        }

        System.out.println("Total Marks: " + sum + "\n Average Marks: " + sum / (arr.length / 2));
    }

}
